src/GuiBase/Viewer/CameraManipulator.cpp

```diff
@@ -61,57 +75,61 @@ void Gui::CameraInterface::resetToDefaultCamera() {
     }
 }
 
-Gui::CameraInterface::~CameraInterface() {}
+Gui::CameraManipulator::~CameraManipulator() {}
 
-void Gui::CameraInterface::resizeViewport( uint width, uint height ) {
+void Gui::CameraManipulator::resizeViewport(uint width, uint height ) {
     m_camera->resize( Scalar( width ), Scalar( height ) );
 }
 
-Core::Matrix4 Gui::CameraInterface::getProjMatrix() const {
+Core::Matrix4 Gui::CameraManipulator::getProjMatrix() const {
     return m_camera->getProjMatrix();
 }
 
-Core::Matrix4 Gui::CameraInterface::getViewMatrix() const {
+Core::Matrix4 Gui::CameraManipulator::getViewMatrix() const {
     return m_camera->getViewMatrix();
 }
 
-void Gui::CameraInterface::setCameraSensitivity( double sensitivity ) {
+void Gui::CameraManipulator::setCameraSensitivity(double sensitivity ) {
     m_cameraSensitivity = sensitivity;
 }
 
-void Gui::CameraInterface::setCameraFov( double fov ) {
+void Gui::CameraManipulator::setCameraFov(double fov ) {
     m_camera->setFOV( fov );
 }
 
-void Gui::CameraInterface::setCameraFovInDegrees( double fov ) {
+void Gui::CameraManipulator::setCameraFovInDegrees(double fov ) {
```

Comment:
```suggestion
void Gui::CameraManipulator::setCameraFovInDegrees(Scalar fov ) {
```
also update the definition in hpp